Index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cosmetics Store</title>
+    <link rel="stylesheet" href="./WelcomePage/WelcomePage.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+      integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <main class="container">
+      <header class="logo">
+        <img src="./Assets/logo.png" alt="logo-cosmetics" width="100%" />
+      </header>
+      <section>
+        <p class="description">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit.
+        </p>
+      </section>
+      <section class="signIn">
+        <input
+          class="inpName"
+          type="text"
+          name="inputName"
+          id="inputName"
+          placeholder="Enter your name"
+        />
+        <button class="btnLogin">Let's go</button>
+      </section>
+    </main>
+  </body>
```

Comment:
Could you please add this here
`<script src="./WelcomePage/WelcomePage.js"></script>`
no body of team linked the page with js